src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+@import './body';
+@import './grade';
+@import './price';
+
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__items {
+  object-fit: contain;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  margin: 0 16px 4px;
+  color: $main-text-color;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), color values should be stored in variables to improve maintainability. Here, `$main-text-color` is used which is good, but for consistency, consider storing `#616070` (line 38) in a variable as well.